examples/purchaseTesterTypescript/Gemfile.lock

```diff
@@ -104,7 +104,7 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
-  cocoapods (~> 1.11, >= 1.11.3)
+  cocoapods
```

Comment:
I faced issues the other day with older versions of cocoapods. They got fixed with the latest one. This guarantees we use the latest one